packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -1000,6 +1000,15 @@ const Account = Backbone.Model.extend(
       );
     },
 
+    /**
+     * Fetch the account's list of settings data.
+     *
+     * @returns {Promise} - resolves with a list of settings objects.
+     */
+    getSettingsData() {
+      return this._fxaClient.account(this.get('sessionToken'));
+    },
```

Comment:
Hey @LZoog, if you rebase this branch against latest `master`, you should be able to get rid of this method and call the `settingsData` method that was added in #1846.